e2e-pw/src/oss/specs/grid-tagging.spec.ts

```diff
@@ -44,17 +44,22 @@ test("grid tagging", async ({ fiftyoneLoader, grid, page, sidebar }) => {
   await sidebar.clickFieldCheckbox("filepath");
   await sidebar.clickFieldCheckbox("tags");
   await grid.scrollBottom();
-  await expect(await grid.locator).toHaveScreenshot("grid-untagged.png", {
-    animations: "allow",
-  });
+  for (let i = 31; i <= 54; i++) {
+    const locator = grid.locator.getByText(`/tmp/${i}.png`);
+    await expect(locator).toBeVisible();
+  }
 
   await grid.run(async () => {
     await grid.actionsRow.toggleTagSamplesOrLabels();
     await grid.tagger.setActiveTaggerMode("sample");
     await grid.tagger.addNewTag("sample", "grid-test");
   });
 
-  await expect(await grid.locator).toHaveScreenshot("grid-tagged.png", {
-    animations: "allow",
-  });
+  for (let i = 31; i <= 54; i++) {
+    const locator = grid.locator.getByText(`/tmp/${i}.png`);
+    await expect(locator).toBeVisible();
+    await expect(
+      locator.locator("..").getByTestId("tag-tags-grid-test")
+    ).toBeVisible();
+  }
```

Comment:
31...54 look like magic numbers, add more context?

Reply:
It happens to be the visible images when calling scrollBottom once. I will add a note in another PR